### ISSUE TITLE
data structure changes

### ISSUE THREAD

**Issue by palica** [haxmeister/funtoo-reporter#28]
Hi Haxmeister!

I am sorry to bother you, but could we change the data that is being send to ES like this?

```
{
        "_index": "funtoo",
        "_type": "report",
        "_id": "",  <= leave this empty and let ES add one
        "_source": {
          "@timestamp": "epoch",
          "system": {
            "data": {
....
              },
            }
          },
          "funtoo-report": {
            "UUID": "uuid",
            "version": "1.2"
          }
        }
}
      
```

the main change being removing the uuid from the _id and adding it to the body. reasoning: currently as the systems report they update the record according to the uuid, but elasticsearch is not able to track changes in time, so we cannot see the settings the systems had 2 weeks ago.

**Comment by haxmeister** [comment]
UUID has now been moved to the report body as follows, also the _id string in the report header is empty.

` "funtoo-report" : {
      "UUID" : "f74ddb0f-57ad-47e8-92ea-466d936b78c5",
      "version" : "1.2"
   },
`

**Comment by palica** [comment]
Addressed in this commit 3d0044c697af23d8cde69268a210d8de14d6e9ad

can be closed.
